src/pages/Login/InputContainer/InputBox/InputBox.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './InputBox.scss';
+
+class InputBox extends React.Component {
+  handleChange = e => {
+    const { value } = e.target;
+    const { name, changeState } = this.props;
+    changeState(name, value);
+  };
+  render() {
+    const { name, imgUrl, type, id, placeholder, goTo } = this.props;
+    return (
+      <form className="inputBox" onSubmit={e => goTo(e)}>
```

Comment:
- event 는 자동으로 인자로 들어갑니다. 따라서 추가적으로 콜백 함수를 생성하실 필요가 없습니다.
```suggestion
      <form className="inputBox" onSubmit={goTo}>
```
- goTo 라는 함수 이름만으로는 이 함수가 어떤 행동을 하는지 정확하게 파악하기 어려운 것 같습니다. 좀 더 구체적으로 이름지어 주세요! ex) `goToMain` 와 같이 어디로 이동하는지 알수있게요!

Reply:
이 conponent를 쓰이는 곳마다  goTo 함수의 역할이 다달라서 정확하게 네이밍을 어떻게해야할지 고민이네요 ! ㅠ